cmd/juju/application/setseries_test.go

```diff
@@ -0,0 +1,90 @@
+// Copyright 2018 Canonical Ltd.
+// Licensed under the AGPLv3, see LICENCE file for details.
+
+package application
```

Comment:
should this not be `application_test`, so that we're only testing input/outputs and not internal?

Reply:
Interesting... it appears that 2/3 of the application test files, don't define the package as application_test....  I moved a few more tests over.